File: Examples/CSharp/ProgrammersGuide/WorkingWithProjects/ReadProjectFiles.cs
using System.IO;
using Aspose.Tasks;
using System;

namespace Aspose.Tasks.Examples.CSharp.WorkingWithProjects
{
    public class ReadProjectFiles
    {
        public static void Run()
        {
            //ExStart: ReadProjectFiles
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Projects();

            Project project = new Project(dataDir + "ReadProjectFiles.mpp");
            //ExEnd: ReadProjectFiles
        }
    }
}

File: Examples/VisualBasic/ProgrammersGuide/WorkingWithProjects/ReadProjectFiles.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Tasks
Imports System

Namespace Aspose.Tasks.Examples.VisualBasic.WorkingWithProjects
    Public Class ReadProjectFiles
        Public Shared Sub Run()
            'ExStart: ReadProjectFiles
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Projects()

            Dim project As New Project(dataDir & "ReadProjectFiles.mpp")
            'ExEnd: ReadProjectFiles
        End Sub
    End Class
End Namespace